common/src/models/magic_wand/magic_wand.c

```diff
@@ -53,36 +53,44 @@ MagicWandResult magic_wand_classify() {
   };
 }
 
+static void print_magic_wand_result(const char* prefix, MagicWandResult res) {
+  printf("%s-- wing: 0x%x, ring: 0x%x, slope: 0x%x, negative: 0x%x\n", prefix,
+         res.wing_score, res.ring_score, res.slope_score, res.negative_score);
+  printf(
+      "%s approx-- wing: 0.%06ld ring: 0.%06ld, slope: 0.%06ld, negative: "
+      "0.%06ld\n",
+      prefix, (int)(*(float*)&res.wing_score * 1000000),
```

Comment:
It looks like it would be more convenient if the MagicWandResult member variables were floats. Any reason not to do that?